audiomentations/core/composition.py

```diff
@@ -125,20 +143,119 @@ class OneOf(BaseCompose):
     # Augment/transform/perturb the audio data
     augmented_samples = augment(samples=samples, sample_rate=16000)
 
+    # Result: 2 or more transforms will be applied from the list of transforms.
+    ```
+
+    """
+
+    def __init__(
+        self, nbr_transforms: float or tuple, transforms, p: float = 1.0
+    ):
+        super().__init__(transforms, p)
+        self.transform_indexes = []
+        self.nbr_transforms = nbr_transforms
+        self.should_apply = True
+
+    def randomize_parameters(self, *args, **kwargs):
+        super().randomize_parameters(*args, **kwargs)
+        self.should_apply = random.random() < self.p
+        if self.should_apply:
+            if type(self.nbr_transforms) == tuple:
+                if self.nbr_transforms[1] is None:
+                    nbr_transforms_to_apply = random.randint(
+                        self.nbr_transforms[0], len(self.transforms)
+                    )
+                else:
+                    nbr_transforms_to_apply = random.randint(
+                        self.nbr_transforms[0], self.nbr_transforms[1]
+                    )
+            else:
+                nbr_transforms_to_apply = self.nbr_transforms
+            all_transforms_indexes = list(np.arange(len(self.transforms)))
```

Comment:
This is no biggie, but using python's internal "range" would also work well here, avoiding the little roundtrip to numpy and back

```suggestion
            all_transforms_indexes = list(range(len(self.transforms)))
```